### ISSUE TITLE
docs/api: Each param on a separate line in source code

### ISSUE THREAD

**Issue by petebacondarwin** [angular/angular#22364]
See @jbogarthyde's comment here: https://github.com/angular/angular/pull/21874#issuecomment-363861807

I think this is something that the Angular core team would need to make a decision upon and implement via a clang rule - cc @IgorMinar or @alexeagle ?

**Comment by alexeagle** [comment]
Do you mean the suggestion that we put each parameter on its own line?
This thing:
https://docs.google.com/document/d/1jDzRbGdh99nzrbo8zhuJ_JsqvOv_AYQYijQaU76Mc14/edit
 ?

The most likely answer here is to switch to Prettier, we can discuss if
you're curious. Gated on getting incremental formatting setup.

**Comment by IgorMinar** [comment]
yeah. this is unrelated to docs/api effort IMO. we do want to reformat our code base, but that's a separate effort and the result of this effort (or lack of it) should in no way influence api docs.

**Comment by jelbourn** [comment]
Closing since it's not clear exactly what this issue is tracking
